src/mono/mono/sgen/sgen-cardtable.c

```diff
@@ -165,6 +165,31 @@ sgen_card_table_wbarrier_range_copy (gpointer _dest, gconstpointer _src, int siz
 	}
 }
 
+// Marks more cards so that it works with remset consistency debug checks
```

Comment:
so is this only enabled in debug builds ?

Reply:
Sort of, only if certain debug flags are passed at runtime via env vars this is used to mark cards when copying objects. It is a conservative approach. It might be useful to backport since there are active crashes there that are not actionable and this PR should at least fix the use of some GC debugging features that might help investigate.